edx_django_utils/management/docs/decisions/0001-user-and-group-management-commands.rst

```diff
@@ -0,0 +1,20 @@
+User and Group Management Commands
```

Comment:
1. Could you add this to the "Decisions" section of the main index page? See https://edx.readthedocs.io/projects/edx-django-utils/en/latest/.  You should be able to test this locally in a virtual env with `make docs`.
2. In the main README, it reads `TODO: Publish to https://edx-django-utils.readthedocs.org.`. Would you mind updating that to say `See the published docs at https://edx-django-utils.readthedocs.org.`.